interface/forms/procedure_order/common.php

```diff
@@ -662,7 +662,7 @@ function sel_related(varname) {
 // Also note the controlling script here runs from interface/patient_file/encounter/.
     let title = '<?php echo xla("Select Diagnosis Codes"); ?>';
     <?php /*echo attr(collect_codetypes("diagnosis", "csv")); */?>
-    dlgopen(top.webroot_url + '/interface/patient_file/encounter/find_code_dynamic.php?codetype=ICD10', '_blank', 985, 750, '', title);
+    dlgopen(top.webroot_url + '/interface/patient_file/encounter/find_code_dynamic.php?codetype=ICD10,LOINC,CPT4', '_blank', 985, 750, '', title);
```

Comment:
LOINC codes are iffy for diagnosis but for sure don't need cpt4. Better to use SNOMED instead

Reply:
I think these are unnecessary. perhaps I mislead. can we have a call?

Reply:
Just checking if there was an outcome on what to include here?

Reply:
I have to have LOINC codes on the procedure to be compliant with FHIR.  I added SNOMED per @sjpadgett's recommendations.  I'm fine with removing SNOMED if we don't need them.  I haven't talked to @sjpadgett about it yet.